plugins/SnapshotPlugin/mock-window.js

```diff
@@ -1,11 +1,27 @@
-/* eslint no-var: off */
+/* eslint no-var: off, prettier/prettier: off */
 /*
  *  mock-window.js
  *
  *  A helper utility meant to simulate a basic window object for ReactDOM usage during snapshot execution.
  */
 
 var orig = {}, listeners = [], nop = function() {};
+var defer = function() {
+	var objPath = arguments;
+	var deferred = function() {
+		var ctx = global;
+		var key = objPath[objPath.length - 1];
+		for(var i = 0; i < objPath.length - 1; i++) {
+			ctx = ctx[objPath[i]]
```

Comment:
Should we be defensive here and bail if `ctx` is undefined at some point. For example, if `defer` is used with a path that doesn't resolve on a particular plaform. Not sure how likely that scenario is but the cost is minimal for the protection.